### PR TITLE
OPENSHIFTP-8: add fips enablement for 4.16 and higher

### PR DESCRIPTION
**Pull request by bawaler**
upgrading the ocp4-helpernode tag version, and adding a reboot to support FIPS cluster creation.

CC: @prb112

### CODE REVIEW

**Comment by prb112** [comment]
Hey @yussufsh this PR is tested on powervs, and puts ocp4-upi-powervm and ocp4-upi-powervs at the same parity for FIPS support. Thanks, Paul

**Comment by yussufsh** [comment]
> Hey @yussufsh this PR is tested on powervs, and puts ocp4-upi-powervm and ocp4-upi-powervs at the same parity for FIPS support. Thanks, Paul

@prb112 In install.tf we already had the restart bastion logic in resource named "fips_bastion_reboot". If you think that is not working and need it this way, remove from install.tf. Either way works for me (not both).

**Comment by prb112** [comment]
Excellent memory Yussuf. Thank you.

**Comment by prb112** [comment]
waiting on the check to verify formatting. It looks good overall.

File: modules/5_install/install.tf [hunk 1]
```diff
@@ -589,11 +590,3 @@ resource "null_resource" "csi_driver_install" {
   }
 }
 
-resource "ibm_pi_instance_action" "fips_bastion_reboot" {
```
Comment (prb112) [inline modules/5_install/install.tf:592]: removed per discussion

**Comment by prb112** [comment]
Hey @yussufsh Rohini has removed and tested the changes. back to you for review, Thanks, Paul

**Comment by yussufsh** [comment]
> Hey @yussufsh Rohini has removed and tested the changes. back to you for review, Thanks, Paul

@prb112 LGTM, please rebase.

**Comment by prb112** [comment]
Hi @yussufsh - Rohini has rebased the code. Back to you, thank you, Paul

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by prb112** [comment]
Hey Yussuf, Thank you, I see the label `do-not-merge/non-mergeable` is still in place. Anything I need to do? Thank you, Paul

**Comment by prb112** [comment]
thank you as always

**Pull request merged**